### PR TITLE
Fix single work scaffold

### PR DESCRIPTION
**Pull request by varffs**
allows single images no gallery, video [placeholder for now until
platform known] and in progress status

### CODE REVIEW

File: lib/meta-boxes.php [hunk 1]
```diff
@@ -119,6 +126,20 @@ function igv_cmb_metaboxes() {
   ) );
 
   $obra_meta->add_field( array(
+    'name'       => __( 'Image', 'cmb2' ),
+    'desc'       => __( 'if single image not slideshow', 'cmb2' ),
+    'id'         => $prefix . 'image',
+    'type'       => 'file',
+  ) );
```
Comment (mixcas) [inline lib/meta-boxes.php]: why not the feat image? might be diff from thumb?

Comment (varffs) [inline lib/meta-boxes.php]: hmm good point. not sure rly. perhaps flexibility is good or too many options confuse the user. what do you think?

Comment (varffs) [inline lib/meta-boxes.php]: certainly the example content has never had different thumbs from the single image

Comment (mixcas) [inline lib/meta-boxes.php]: Yeah, i think it's confusing and yeah, designs match thumb with that image.

File: lib/meta-boxes.php [hunk 2]
```diff
@@ -119,6 +126,20 @@ function igv_cmb_metaboxes() {
...
+
+  $obra_meta->add_field( array(
+    'name'       => __( 'Video', 'cmb2' ),
+    'desc'       => __( 'if video not slideshow. [vimeo embed id or ???]', 'cmb2' ),
```
Comment (mixcas) [inline lib/meta-boxes.php:130]: im thinking embed code or link
if link, we can convert it to embed code with oembed
either way we can later wrap it with a responsive container

thoughts?

Comment (varffs) [inline lib/meta-boxes.php:130]: yh tbh i need to email about this feature. wasnt ever in any of the pdfs but obv a vimeo/youtube embed is very easy. I prefer IDs to urls and oembed tbh but isnt that different. I guess leave this as placeholder and do a video feature branch when we kno

Comment (mixcas) [inline lib/meta-boxes.php:130]: why do u hate oembed?

**Pull request merged**